src/components/CanvasWrapper/BackdropToggleButton.tsx

```diff
@@ -0,0 +1,75 @@
+import { Radio } from "antd";
+import React, { ReactElement, ReactNode, useContext } from "react";
+
+import { TabType } from "../../colorizer";
+import { useViewerStateStore } from "../../state";
+import { VisuallyHidden } from "../../styles/utils";
+import { makeLinkStyleButton } from "./utils";
+
+import { AppThemeContext } from "../AppStyle";
+import { ImageToggleButton } from "../Buttons/ImageToggleButton";
+
+export default function BackdropToggleButton(): ReactElement {
+  const theme = useContext(AppThemeContext);
+
+  const dataset = useViewerStateStore((state) => state.dataset);
+  const backdropKey = useViewerStateStore((state) => state.backdropKey);
+  const backdropVisible = useViewerStateStore((state) => state.backdropVisible);
+  const setOpenTab = useViewerStateStore((state) => state.setOpenTab);
+  const setBackdropKey = useViewerStateStore((state) => state.setBackdropKey);
+  const setBackdropVisible = useViewerStateStore((state) => state.setBackdropVisible);
+
+  // Tooltip shows current backdrop + link to viewer settings
+  const backdropTooltipContents: ReactNode[] = [];
+  const backdropData = dataset?.getBackdropData();
+  const hasBackdrops = backdropData !== undefined && backdropData.size > 0;
+
+  if (!hasBackdrops) {
+    backdropTooltipContents.push(<span key="no-backdrops">(No backdrops available)</span>);
+  } else if (backdropData.size === 1) {
+    // If only one backdrop is available, just show its name
+    backdropTooltipContents.push(
+      <span key="backdrop-name" style={{ color: theme.color.text.button }}>
+        {backdropKey && backdropData.get(backdropKey)?.name}
+      </span>
+    );
+  } else {
+    // Create a radio button for each backdrop
```

Comment:
New feature! When multiple backdrops are selected, the tooltip shows a radio that lets users switch between them.
<img width="595" height="256" alt="Image" src="https://github.com/user-attachments/assets/db728862-066f-42bb-8b49-7bab439aee0d" />